README.md

```diff
@@ -1,7 +1,104 @@
+
+We generally follow [the Atlassian model](https://www.atlassian.com/agile/project-management/epics-stories-themes), with some naming changes to avoid confusion in [our tooling](#Tools).
+
+Theme -> Initiative -> Milestone -> Epic -> Issue
+
+### Theme
+Themes are large focus areas than span the Mozilla organization and are pertinent and addressable by the Taskcluster team.
+
+For Taskcluster @ Mozilla, we are currently tracking the following themes:
+* Making Data Actionable
+* Ease of use
+* Promote external adoption
+* Cost Reduction
+
+### Initiative
+An _initiative_ is a collection of one or more milestones that, taken together, address one or more themes. Depending on the project area, an initiative can be a thin wrapper around a single milestone if that milestone is high value and self-contained.
+
+### Milestone
+The _milestone_ is the goal for a sprint and, as such, should be able to be completed during the sprint duration made up of a small collection of epics. This can be a thin wrapper around a single epic if the user story is large and/or important enough. Milestones can address one or more themes.
+
+### Epic
+An _epic_ a large user story that describes plainly a feature of the Taskcluster system. Epics can address multiple themes. User stories are an important aspect of agile development, but we've chosen to map them to Epics in our usage to make them trackable at the correct level using [our tooling](#Tools).
+
+### Issue
+An _issue_ is a single, cohesive task as represented by a Github issue. Larger issues should be decomposed into smaller issues. No single issue should taken more than 1 engineer week to accomplish, modulo time to review and deploy. Issues should address a single theme.
+
+A well-defined developmemt issue is the basis for all of our estimation. It cannot be overstated how important it is to make sure issues are decomposed to the point where they can be estimated accurately. It underpins _everything_ else.
+
 ## Tools
+All Taskcluster work for sprints is tracked via issues in Github.
+
+Sometimes bugs from [Bugzilla](https://bugzilla.mozilla.org) will be part of our sprint. When this happens we will create a corresponding issue or Epic in Github and reference the original bug.
+
+We are using [Zenhub](https://www.zenhub.com/) to manage our agile process. Zenhub layers much-needed functionality on top of Github, namely:
+* dependency tracking
+* estimate tracking
+* milestones for tracking sprint objectives
+* burndown charts
+
+## Roles
+We have three defined roles in our agile process:
+### 1. Product owner
+The _Product owner_ manages the product backlog and keeps the rest of the team working on the most important thing. They have the final decision on matters of scope. They can also change the scope of the current milestone or, at their discretion,  end the sprint early.
```

Comment:
double space before "end the sprint early"

Reply:
Fixed.